src/Module.Server/Common/CrpgBattleScoreboardData.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Globalization;
+using Crpg.Module.Common;
+using TaleWorlds.Localization;
+using TaleWorlds.MountAndBlade;
+
+using TaleWorlds.MountAndBlade.Diamond.MultiplayerBadges;
+
+namespace Crpg.Module.Modes.Skirmish;
+
+internal class CrpgBattleScoreboardData : IScoreboardData
+{
+    public MissionScoreboardComponent.ScoreboardHeader[] GetScoreboardHeaders()
+    {
+        return new MissionScoreboardComponent.ScoreboardHeader[]
+        {
+            new("ping", missionPeer => TaleWorlds.Library.MathF.Round(missionPeer.GetNetworkPeer().AveragePingInMilliseconds).ToString(), _ => "BOT"),
+            new("level", missionPeer => missionPeer.GetComponent<CrpgPeer>().User?.Character.Level.ToString() ?? string.Empty, _ => string.Empty),
```

Comment:
NIT: Why not 0 or 1 as default value instead of an empty string?

Reply:
oh, it's because of Bots right?

Reply:
If for some reason the User is not synced , i do not want to display a wrong value